insights/specs/insights_archive.py

```diff
@@ -173,6 +173,7 @@ class InsightsArchiveSpecs(Specs):
     modinfo_veth = simple_file("insights_commands/modinfo_veth")
     modinfo_vmxnet3 = simple_file("insights_commands/modinfo_vmxnet3")
     mokutil_sbstate = simple_file("insights_commands/mokutil_--sb-state")
+    mpirun_version = simple_file("insights_commands/usr.local.bin.mpich_--version")
```

Comment:
@mohitkumarrh looks like you have a typo in the spec name.  The `mangle` command provides the correct path in an archive:
```
$ mangle "/usr/local/bin/mpirun --version"
insights_commands/usr.local.bin.mpirun_--version
```

Reply:
@bfahr - Thanks for the review, Yes i made a typo mistake in mangle command. I'll correct this typo and update the PR